### PR TITLE
Make package.json scripts work on Windows

### PR DESCRIPTION
**Pull request by c-w**
Currently there are several bashisms in the various package.json scripts which makes it impossible to work on boardgame.io natively on Windows without using a Linux emulation layer such as WSL. This commit replaces the bashisms with cross-platform Node commands.

#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] Test coverage is 100% (or you have a story for why it's ok).

### CODE REVIEW

**Comment by nicolodavis** [comment]
Thanks for taking the time to do this!!  Will take a closer look in a few days (busy this week).

File: package.json [hunk 1]
```diff
@@ -110,6 +113,7 @@
     "lint-staged": "^8.1.0",
     "node-persist": "^3.0.4",
     "nodemon": "^1.18.9",
+    "npm-run-all": "^4.1.5",
```
Comment (ksjogo) [inline package.json:116]: Does one need npm-run-all and concurrently? They seem to cover pretty much the same functionality as far as I can tell.

File: scripts/changelog.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// Generates docs/CHANGELOG.md
+// Run this right after the npm version command.
+
+const { EOL } = require('os');
+const shell = require('shelljs');
```
Comment (ksjogo) [inline scripts/changelog.js:5]: Why have I not been using shelljs before? :D

File: package.json (hunk 1 above)
Comment (ksjogo) [inline package.json:116]: Does one need npm-run-all and concurrently? They seem to cover pretty much the same functionality as far as I can tell.

Comment (c-w) [reply package.json:115]: Replaced `concurrently` with `run-p` in 7a0c04f since `concurrently -m1` didn't approximate `run-s` for me.

**Comment by nicolodavis** [comment]
@c-w Made a few changes.  Let me know if they look good to you.

The main addition is to run "build" in prestart so you can just do:
```
npm install
npm start
```

when you first clone the repo.

**Comment by c-w** [comment]
@nicolodavis Changes LGTM.

**Comment by delucis** [comment]
Thanks Clemens!

**Pull request merged**